test/test_syscalls.c

```diff
@@ -201,7 +201,7 @@ s();
 s();
 	(void) lseek(fd, 0, SEEK_SET);
 s();
-	(void) read(fd, buffer, BUF_SIZE);
+	int rb = read(fd, buffer, BUF_SIZE);
```

Comment:
I don't see how it can affect Coverity...

Reply:
However, it does ...

Reply:
It is not my fault that Coverity works like this ;-)